app/src/main/java/co/nimblehq/compose/crypto/ui/components/chartintervals/ChartIntervalsButton.kt

```diff
@@ -0,0 +1,84 @@
+package co.nimblehq.compose.crypto.ui.components.chartintervals
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.compose.crypto.ui.theme.Color
+
+@Composable
+fun ChartIntervalsButtonGroup(
+    modifier: Modifier,
+    onIntervalChanged: (TimeIntervals) -> Unit
+) {
+
+    val selectedColor = remember { mutableStateOf(0) }
+    Row(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.spacedBy(14.dp)
+    ) {
+        val listOfInterval = listOf(
+            TimeIntervals.ONE_DAY,
+            TimeIntervals.ONE_WEEK,
+            TimeIntervals.ONE_MONTH,
+            TimeIntervals.ONE_YEAR,
+            TimeIntervals.FIVE_YEAR
+        )
+
+        listOfInterval.forEachIndexed { index, interval ->
+            val backgroundColor = if (selectedColor.value == index) {
+                Color.CaribbeanGreen
+            } else {
+                androidx.compose.ui.graphics.Color.Transparent
```

Comment:
Shall we add it in `Color.kt` then we can get rid of `androidx.compose.ui.graphics` here.

Reply:
@Wadeewee Yeah we can, but it will redundant imo 😄